src/store/modules/layers.store.js

```diff
@@ -220,22 +228,16 @@ const actions = {
      * Will set the background to the given layer (or layer ID), but only if this layer's
      * configuration states that this layer can be a background layer (isBackground flag)
      *
-     * @param {String | AbstractLayer} bgLayer Either the background layer id or an AbstractLayer
-     *   object
+     * @param {String | null} bgLayerId The background layer id object
      * @param {string} dispatcher Action dispatcher name
      */
-    setBackground({ commit, getters }, { bgLayer, dispatcher }) {
-        const layerIdOrObject = bgLayer
-        let futureBackground
-        if (typeof layerIdOrObject === 'string') {
-            futureBackground = getters.getLayerConfigById(layerIdOrObject)
-        } else if (layerIdOrObject instanceof AbstractLayer) {
-            futureBackground = getters.getLayerConfigById(layerIdOrObject.id)
+    setBackground({ commit, getters }, { bgLayerId }) {
+        if (bgLayerId == null) {
```

Comment:
triple `===` please 😉

Reply:
Yes sir! 😉
  Can't we enforce this with eslint?

Reply:
Yes sir 😉
Can't we enforce this with eslint?

Reply:
we could maybe... https://eslint.org/docs/latest/rules/eqeqeq
 there are rarely cases where we want to double equals, but it would make sense to then disable the rule

Reply:
As discussed on Slack, I will maybe check this with another branch, as 1) adding the rule would probably require quite many adjustments throughout the code base and 2) the rule should actually already be enabled by default